source/uwp/Renderer/lib/Util.cpp

```diff
@@ -370,104 +367,110 @@ HRESULT GenerateActionProjection(
     {
-        case ActionType::OpenUrl:
-            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveOpenUrlAction>(projectedAction,
-                std::AdaptivePointerCast<AdaptiveSharedNamespace::OpenUrlAction>(action)));
-            break;
-        case ActionType::ShowCard:
-            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveShowCardAction>(projectedAction,
-                std::AdaptivePointerCast<AdaptiveSharedNamespace::ShowCardAction>(action)));
-            break;
-        case ActionType::Submit:
-            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveSubmitAction>(projectedAction,
-                std::AdaptivePointerCast<AdaptiveSharedNamespace::SubmitAction>(action)));
-            break;
-        case ActionType::Custom:
-            RETURN_IF_FAILED(std::AdaptivePointerCast<CustomActionWrapper> (action)->GetWrappedElement(projectedAction));
-            break;
-        default:
-            return E_UNEXPECTED;
-            break;
+    case ActionType::OpenUrl:
+        RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveOpenUrlAction>(projectedAction,
+                                                                                                                                                                     std::AdaptivePointerCast<AdaptiveSharedNamespace::OpenUrlAction>(action)));
+        break;
+    case ActionType::ShowCard:
+        RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveShowCardAction>(projectedAction,
+                                                                                                                                                                        std::AdaptivePointerCast<AdaptiveSharedNamespace::ShowCardAction>(action)));
+        break;
+    case ActionType::Submit:
+        RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveSubmitAction>(projectedAction,
+                                                                                                                                                                    std::AdaptivePointerCast<AdaptiveSharedNamespace::SubmitAction>(action)));
+        break;
+    case ActionType::Custom:
+        RETURN_IF_FAILED(std::AdaptivePointerCast<CustomActionWrapper>(action)->GetWrappedElement(projectedAction));
+        break;
+    default:
+        return E_UNEXPECTED;
+        break;
     }
 
     return S_OK;
-} CATCH_RETURN;
+}
+CATCH_RETURN;
 
 HRESULT GenerateColumnsProjection(
-    const std::vector<std::shared_ptr<AdaptiveSharedNamespace::Column>>& containedElements,
-    ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveNamespace::IAdaptiveColumn*>* projectedParentContainer) noexcept try
+        const std::vector<std::shared_ptr<AdaptiveSharedNamespace::Column>> &containedElements,
+        ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveNamespace::IAdaptiveColumn *> *projectedParentContainer) noexcept try
```

Comment:
> *> * [](start = 94, length = 5)

Why are we moving these * over? Most of our other code is consistent with the style before this change.

Reply:
I'm not sure why it's saved like that for this file

Reply:
If you don't have any changes you actually want to make to this file, consider checking it out from master to get back to where it started.

---
In reply to: [188026941](https://github.com/Microsoft/AdaptiveCards/pull/1437#discussion_r188026941) [](ancestors = 188026941)